.github/workflows/create-release.yml

```diff
@@ -8,7 +8,8 @@ on:
 jobs:
   create-release:
     runs-on: ubuntu-latest
-    if: ${{ github.ref == 'refs/heads/main' && github.actor == 'liatrio-otel-collector-release[bot]' && startsWith(github.event.head_commit.message, 'Prepare liatrio-otel for version') }}
+    if: ${{ github.ref == 'refs/heads/main' && github.actor == 'liatrio-otel-collector-release[bot]' && startsWith(github.event.head_commit.message, 'Prepare
+      liatrio-otel for version') }}
```

Comment:
This is bad syntax please revert

Reply:
The precommit hook does this. But linting in vscode was giving a red squiggly. So you have to commit with `--no-verify` 🥴

Reply:
Fixed